src/v2/models/AppState.js

```diff
@@ -155,11 +155,9 @@ export default Model.extend({
     const previousRawState = this.has('idx') ? this.get('idx').rawIdxState : null;
 
     const identicalResponse = _.isEqual(
-      _.nestedOmit(transformedResponse.idx.rawIdxState, ['expiresAt', 'refresh', 'stateHandle']),
-      _.nestedOmit(previousRawState, ['expiresAt', 'refresh', 'stateHandle']));
-    const isSameRefreshInterval = _.isEqual(
-      _.nestedOmit(transformedResponse.idx.rawIdxState, ['expiresAt', 'stateHandle']),
-      _.nestedOmit(previousRawState, ['expiresAt', 'stateHandle']));
```

Comment:
why those changes?

Reply:
since stateHandle can change between the remediations we do now want to check for it, a response can not be identical if `stateHandle` changed

Reply:
Even stateHandle changed but remediation array isn't changed, I guess we don't need re-render, correct?

Reply:
That is right, but i am using this function to update the AppState and not to re-render, At least till we fix polling. that would be right after this PR.

Updating AppState without fixing polling is causing multiple polling views to fail..